debug-utils/src/index.ts

```diff
@@ -0,0 +1,14 @@
+export * from './debugging'
+import { Config } from '@algorandfoundation/algokit-utils'
+import { DebugParams } from '@algorandfoundation/algokit-utils/types/debugging'
+import { persistSourceMaps, simulateAndPersistResponse } from './debugging'
+
+// Automatically register debug handlers upon import
+Config.registerDebugHandler(async (params: DebugParams) => {
```

Comment:
@neilcampbell how about Config.configure method taking an extra "addons" key where value is list of callbacks to invoke? The main thing i wanted to avoid is to force user doing extra actions on top of indicating via debug flag that user wants to enable this functionality but explicit addons key might separate the concerns a bit in this case

Reply:
another thing to keep in mind is the fact that after this potential split of the package, there will be slight discrepancy on utils-py because we wouldn't need to do the same split there, hence why I tried to keep Config.configure interface unchanged and only require to do the import of the package in case utils-ts is used and debug utils are needed

Reply:
Yeah for sure that's something to consider. Maybe we can do a quick design pairing session.